v2/azure-arm.yaml

```diff
@@ -2779,6 +2779,8 @@ objectModelConfiguration:
       PrivateEndpoint:
         $export: true
         $supportedFrom: v2.0.0
+        $manualConfigs:
+          - PrimaryNicPrivateIPAddress
```

Comment:
minor, not consistent casing of property name. I think we want:
`PrimaryNicPrivateIpAddress` -- note that this isn't the recommended Go casing but I think it's the casing that other properties use because we build their names from JSON so it's consistent with our other stuff I think